### DISCUSSION TITLE
Tags to facilitate faster use

### DISCUSSION THREAD

**Discussion by mfpaim** [docmost/docmost#193] [Ideas]
Hi Doc team,

Thanks for your working and generosity delivering such a great tool. 

I am becoming a heavy user and bringing more people onboard. During the use I was wondering if we could "tag" pages and filter by them it would be amazing. 

For example, it could used to put peoples names, teams, start and end dates, status and other millions of uses. 

Maybe you could consider this suggestion for the future.

Thanks a lot and congrats for the Docmost!

MP

**Comment by Julrob199** [comment]
Something like in joplin would be cool (left bar shows all tags), but additional with hierarchical structure of tags.

And on top of that it would be nice, not to define tags in the markdown area with #tag, but that there is a form (like in joplin) to define the tag and then it is written to yaml or something.

**Comment by saimana** [comment]
Absolutely great system. But tagging would be great additional layer of identification of a pages.

**Comment by Githki** [comment]
Tagging would be great, but then it should be combined with e.g. queries like in Logseq. This could provide a section that is automatically growing with the added content (based on tags or other properties). My use case would be to group notes e.g. in a list for a specific tag spread across various pages (-> pages with mixed topics) to get a quick overview for a topic I am working on.

**Comment by mustafa-yilmaz** [comment]
User groups could also be tagged in order to manage (track, notify) people with group-based tasks.

**Comment by Philipinho** [comment]
Labels/tags will be available in the next version release.

PR: https://github.com/docmost/docmost/pull/2188.

**Comment by dasbootbee** [comment]
Thanks for adding this feature.

How do I see a list of all labels?  The only place I see reference to them is in the page info screen on an individual page.
Are labels going to be searchable at some point?
